pkg/cache/v3/linear.go

```diff
@@ -84,7 +83,8 @@ func WithInitialResources(resources map[string]types.Resource) LinearCacheOption
 	return func(cache *LinearCache) {
 		cache.resources = resources
 		for name := range resources {
-			cache.versionVector[name] = 0
+			// ToDo: ordering of WithVersionPrefix and WithInitialResources
```

Comment:
Indeed. It's pretty easy to handle by moving the code setting the version to `NewLinearCache`, after parsing the options.

Reply:
updated to be in the constructor now